nixpkgs_review/report.py

```diff
@@ -63,15 +63,16 @@ def write_error_logs(attrs: List[Attr], directory: Path) -> None:
                 symlink_source.unlink()
             symlink_source.symlink_to(attr.path)
 
-        if attr.path is not None:
+        if attr.drv_path or attr.path is not None:
+            path = attr.drv_path or attr.path or ""
```

Comment:
If path is empty we should not run nix log.

Reply:
I just left this because mypy was complaing that path is Optional[String]...

Reply:
Also this approach wont work as both are always set...